src/Discord.Net/Entities/Guilds/IGuild.cs

```diff
@@ -0,0 +1,90 @@
+// todo: impl, docs
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Wumpus;
+using Wumpus.Entities;
+
+namespace Discord
+{
+    public interface IGuild : ISnowflakeEntity, IDeletable
+    {
+        // todo: IAudioClient
+        string Name { get; }
+        ulong? ApplicationId { get; }
+        ulong EveryoneRoleId { get; }
+        ulong OwnerId { get; }
+
+        int AFKTimeout { get; }
+        bool IsEmbeddable { get; } //review: is this still applicable
+        string VoiceRegionId { get; }
+
+        DefaultMessageNotifications DefaultMessageNotifications { get; }
+        MfaLevel MfaLevel { get; }
+        VerificationLevel VerificationLevel { get; }
+        ExplicitContentFilter ExplicitContentFilter { get; }
+
+        string? IconId { get; }
+        string? IconUrl { get; }
+        string? SplashId { get; }
+        string? SplashUrl { get; }
+
+        bool Attached { get; } // review: how to make this more consumable
```

Comment:
How do you mean?

Reply:
Seems like it COULD be useful, but probably only if it's named something a little more obvious like `IsAttached` and comes along with `Attached` and `Detached` events. Or maybe `IsConnected`?

Reply:
Agree that at least the name needs a change: Design guidelines say that boolean property/method names should begin with a verb.

Reply:
```suggestion
        bool IsAttached { get; } // review: how to make this more consumable
```
I think attached (with proper docs explaining what it is later on) is less ambiguous than `IsConnected`